pkg/virt-launcher/virtwrap/util/sanitize.go

```diff
@@ -21,8 +21,18 @@ package util
 
 import (
 	"strings"
+
+	"kubevirt.io/kubevirt/pkg/api/v1"
 )
 
-func SanitizeDNSLabel(value string) string {
-	return strings.Split(value, ".")[0]
+// Get sanitized hostname according to DNS label rules
+// In case vmi.Spec.Hostname has a value then it already passed
+// the webhook validation for DNS label rules.
+func SanitizeHostname(vmi *v1.VirtualMachineInstance) string {
```

Comment:
you will have to move this function to something like `pkg/util`. If you move it to `virt-launcher`, then this will pull in libvirt-cbindings which has the consequence that virt-controller starts requiring libvirt libraries.

That is also the reason why virt-controller never started in the last ci-run.

Reply:
I started that with pkg/util and removed it,  will bring it back now.  we didn't seem to have a util package up until now